src/Dapr.AzureFunctions.Extension/Bindings/DaprPublishAttribute.cs

```diff
@@ -15,6 +15,12 @@ namespace Dapr.AzureFunctions.Extension
     [Binding]
     public class DaprPublishAttribute : DaprBaseAttribute
     {
+        /// <summary>
+        /// Gets or sets the pub/sub name to publish to.
+        /// </summary>
+        [AutoResolve]
+        public string? PubSubName { get; set; }
```

Comment:
Elsewhere it was mentioned that dapr uses a default pub/sub name of "pubsub". If that's the case, should we default initialize this property to "pubsub"?

Reply:
Thanks, that answers my question. We should then leave this as-is.

Reply:
I really don't like that idea. We've made these things explicit everywhere else (ex state store name in this repo is also explicit). I think in practice this is something users have to care about.

Reply:
Sorry, what idea specifically do you not like (are you responding to my original suggestion that I've since retracted)?

Reply:
I think we're on the same page. I didn't want to bake a default pub/sub name into the codez.